### DISCUSSION TITLE
Certificato delle Competenze

### DISCUSSION THREAD

**Discussion by lucacaputo09** [iisgiua/giuaschool#692] [Richieste di aiuto] [answered]
Salve, potete dirmi come funziona in  classe 2ª il certificato delle Competenze?

**Comment by trinko** [comment, marked as answer]
Il registro giua@school genera il certificato delle competenze nello scrutinio finale della classe seconda.

Tieni presente che:
- il registro attualmente è progettato per l'uso nella scuola superiore, per cui viene generato solo il certificato al termine dell'assolvimento dell'obbligo di istruzione (allo scrutino della seconda).
- è in corso un'aggiornamento per implementare il nuovo certificato delle competenze (decreto n. 14 del 30 gennaio 2024)
